src/tests/sqlsmith/src/sql_gen/query.rs

```diff
@@ -195,7 +195,7 @@ impl<R: Rng> SqlGenerator<'_, R> {
     }
 
     fn gen_select_list(&mut self, num_select_items: usize) -> (Vec<SelectItem>, Vec<Column>) {
-        let can_agg = self.flip_coin();
+        let can_agg = self.should_generate("agg");
```

Comment:
Shall we add some `const FEAT_KEY_AGG: String = ...` things or an auto-generated `enum`?

Reply:
Good suggestion. Suggest enum for now, until we have a more concrete form for the configurations. Otherwise, we may need to refactor the macro several times.